src/components/Landing/Landing.jsx

```diff
@@ -1,18 +1,34 @@
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
+// Landing is the initial page that users will see when they enter the application
 function Landing() {
+  // User-Input state
   const [destination, setDestination] = useState('');
+  const [searchedDestination, setSearch] = useState('');
```

Comment:
For convention and easier understanding your getter and setter should be named the same. E.g. search, setSearch--so if you're down working in the code and you see searchedDestination you can just use setSearchedDestination.